react/src/screens/BookDetail/index.js

```diff
@@ -32,9 +33,29 @@ class BookDetail extends Component {
   }
 }
 
+BookDetail.propTypes = {
+  detailState: PropTypes.shape({
+    buttonDisabled: PropTypes.bool,
+    bookState: PropTypes.string,
+    returnBefore: PropTypes.string
+  }),
+  currentBook: PropTypes.shape({
+    image_url: PropTypes.string,
+    title: PropTypes.string,
+    author: PropTypes.string,
+    year: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    genre: PropTypes.string,
+    description: PropTypes.string
+  }),
+  loading: PropTypes.bool,
+  dispatch: PropTypes.func
```

Comment:
This `dispatch` is not being used, no need to declare it in `propTypes` or destructuring